flexget/plugins/modify/set_field.py

```diff
@@ -34,7 +33,7 @@ def modify(self, entry, config, errors=True):
         """This can be called from a plugin to add set values to an entry"""
         for field in config:
             # If this doesn't appear to be a jinja template, just set it right away.
-            if not isinstance(config[field], basestring) or '{' not in config[field]:
+            if not isinstance(config[field], str) or '{' not in config[field]:
```

Comment:
Doesn't this remove unicode support? Why swtich from basestring -> str, we are not yet python3? :)

Reply:
It's a redundant import actually. Because of `from builtins import *`, `str` is equivalent to `basestring`.

Source: https://python-future.org/compatible_idioms.html#basestring

Reply:
Actually, 'str' is equivalent to 'unicode' on python 2 (because of the `from builtins import *`.) We don't actually allow byte strings in any of these places.